### DISCUSSION TITLE
⚠️About OpenAI banned accounts

### DISCUSSION THREAD

**Discussion by obiscr** [obiscr/ChatGPT#115] [Announcements]
As of 2023/03/15 - [OpenAI has deactivated some accounts](https://github.com/acheong08/ChatGPT/issues/1158). We recommend not using ChatGPT within the plugin for now, not sure if this has anything to do with it for now.

Just in case, we recommend using GPT 3.5 Turbo inside the plugin to avoid being banned.

Please also do not share your API Key for others to use. From what I understand, this behavior may also lead to account deactivation.

Note that this is only a guess at this point to minimize the damage and not to determine exactly what caused the account to be banned.

**Comment by ghost** [comment]
W
